packages/grid/data-grid/src/DataGrid.test.tsx

```diff
@@ -25,7 +25,7 @@ describe('<DataGrid />', () => {
         brand: 'Puma',
       },
     ],
-    columns: [{ field: 'brand', width: 100 }],
+    columns: [{ field: 'brand', width: 180 }],
```

Comment:
Would this be simpler? 100 is the default for all the types.

```suggestion
    columns: [{ field: 'brand' }],
```